### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by navierula** [1init/pwp-capstones#1]
Valid Python Code (4/4)
Code included runs without any errors.

Implementation of Project Requirements (4/4)
Code produces the suite of functions and classes required of it and calls them in an appropriate order.

Software Architecture (4/4)
Code is separated into distinct classes and functions each which are invoked for their own purposes. 

Uses Python Language Features (4/4)
Code uses language features appropriately. If a task can be solved with a Python language feature, it is.

-

You did an excellent job! I also liked the way you represented your output - very creative! 🎨

**Comment by 1init** [comment]
> Valid Python Code (4/4)
> Code included runs without any errors.
> 
> Implementation of Project Requirements (4/4)
> Code produces the suite of functions and classes required of it and calls them in an appropriate order.
> 
> Software Architecture (4/4)
> Code is separated into distinct classes and functions each which are invoked for their own purposes.
> 
> Uses Python Language Features (4/4)
> Code uses language features appropriately. If a task can be solved with a Python language feature, it is.
> 
> 
> You did an excellent job! I also liked the way you represented your output - very creative!

I am not sure how to use Github i normally use a sftp server with put and get commands and ccrypt to encrypt files up there.
Can i ask was those comments at my project i submitted?
are you from CodeAcademy?
Can i ask did i pass?

**Comment by navierula** [comment]
Yes, those were the comments from the project you submitted and I am from Codecademy. You can assume you have passed.

**Comment by 1init** [comment]
> Yes, those were the comments from the project you submitted and I am from Codecademy. You can assume you have passed.

thank you so much cant wait to see the certificate hopefully get it soon to slap on the desk of my manager )))
